problemsets-mat223/modules/module1.tex

```diff
@@ -423,7 +423,7 @@
 	\end{tikzpicture}
 \end{center}
 
-In conjunction with the standard coordinate system, there are also \emph{standard basis vectors}.
+In conjunction with the standard coordinate system, there are also \emph{standard basis vectors}\index[definitions]{standard basis vectors}\index{standard basis vectors}\index[definitions]{vector!standard basis}\index{vector!standard basis}.
```

Comment:
`standard basis vectors` -> `standard basis`

What are you hoping people get out of `vector!standard basis`? When someone is wondering what the standard basis is, are they going to think "Oh, I should think about vectors," go to `V` in the index, and then look for a sub-category of `standard basis`?  When you're deciding whether something should be a subcategory, make sure to think about how a student would actually try to look the thing up. An index that's too big won't be used.